File: code/05198.pas
program Bai198;
  var n, s, n2, c: integer;
begin
  readln(n);
  s := 0;
  n2 := n;
  while n > 0 do 
      begin
      c := n mod 10;
      s := s*10 + c;
      n := n div 10;
      end;
      if s = n2 then writeln('YES') else  write( 'NO');
end.

File: code/05198.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,n2,s,c;
    cin>>n;
    n2=n;s=0;
    while (n>0){
        c=n%10;
        s=s*10+c;
        n=n/10;
    }
    if (s==n2)
        cout<<"YES";
    else cout<<"NO";
    return 0;
}